<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteAccountClass</class>
 <widget class="QMainWindow" name="DeleteAccountClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>501</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DeleteAccount</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_delete_acc">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(60, 60, 60);</string>
    </property>
    <property name="text">
     <string>Zatwierdz usuniecie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_iban_to_delete">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(60, 60, 60);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_aproved">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_more_aproved">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>269</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time_display_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>P$M</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_display_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Menu usuniecia klienta</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_display_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Wprowadz dane usuwanego konta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_display_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Has≈Ço administratora:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_display_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>IBAN:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_display_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>259</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Potwierdzam ze wlasciciel konta splacil wszystkie </string>
    </property>
   </widget>
   <widget class="QLabel" name="time_display_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>zoobowiazania zwiazane z kontem</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_display_10">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Wlasciciel konta zrozumial i zgadza sie z postanowieniami</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_display_11">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>dotyczacymi ochrony danych osobowych</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_out_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255,0,0);</string>
    </property>
    <property name="text">
     <string>Anuluj</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
